prow/prstatus/prstatus.go

```diff
@@ -176,14 +177,28 @@ func (da *DashboardAgent) HandleUserDashboard(queryHandler PullRequestQueryHandl
 				serverError("Error with getting user login", err)
 				return
 			}
-			// Saves login
+			// Saves login. We saves the login under 2 cookies. One for the use of client to render the
```

Comment:
saves -> save. we should also make it clear that the one the client sees is *only* your github username, not any authentication.